tests/bsim/bluetooth/host/gatt/settings_clear/test_scripts/run_settings_clear.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+set -eu
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+test_name="$(guess_test_long_name)"
+simulation_id=${test_name}
+
+verbosity_level=2
+
+EXECUTE_TIMEOUT=120
+
+SIM_LEN_US=$((2 * 1000 * 1000))
+
+test_exe="${BSIM_OUT_PATH}/bin/bs_${BOARD_TS}_${test_name}_prj_conf"
+
+cd ${BSIM_OUT_PATH}/bin
+
+Execute ./bs_2G4_phy_v1 -v=${verbosity_level} -s=${simulation_id} -D=2 -sim_length=${SIM_LEN_US} $@
+
+Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=0 -rs=420 -testid=server \
+		-flash="${simulation_id}_server.settings" -flash_rm -RealEncryption=1
```

Comment:
Maybe a stupid question, but why keep the flash in a file in the disk if you are removing it after terminating the execution? (if you don't set any of the -flash options, the flash is kept by default in the linux process heap and not mapped to the linux filesystem.

Reply:
There is no stupid questions! :)
And indeed, there is no reasons to keep the file in the disk. It's my bad, I thought for some reasons that it was needed for using settings in BabbleSim 😅